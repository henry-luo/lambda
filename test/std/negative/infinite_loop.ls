// Test: Infinite Loop (Should Timeout)
// Category: negative
// Type: negative
// Expected: timeout

// This test should hang and be killed by timeout
let x = 0
x
while true {
    x = x + 1
}
