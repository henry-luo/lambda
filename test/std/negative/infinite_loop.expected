09:15:53 [NOTICE] Start transpiling test/std/negative/infinite_loop.ls...
09:15:53 [ERROR] Syntax tree has errors.
09:15:53 [ERROR] Syntax error at Ln 8, Col 1 - 8, Col 11: ERROR
09:15:53 [ERROR] ERROR node at Ln 8, Col 1 - 8, Col 11
09:15:53 [ERROR] Syntax error at Ln 9, Col 5 - 9, Col 8: ERROR
09:15:53 [ERROR] ERROR node at Ln 9, Col 5 - 9, Col 8
09:15:53 [ERROR] Error: Failed to compile the function.
##### Script 'test/std/negative/infinite_loop.ls' executed: #####
null
